bento_beacon/config_files/config.py

```diff
@@ -145,14 +157,14 @@ class Config:
     KATSU_PUBLIC_CONFIG_ENDPOINT = "/api/public_search_fields"
     KATSU_INDIVIDUAL_SCHEMA_ENDPOINT = "/api/schemas/phenopacket"
     KATSU_EXPERIMENT_SCHEMA_ENDPOINT = "/api/schemas/experiment"
-    KATSU_BEACON_SEARCH = "/api/beacon_search"
+    KATSU_BEACON_SEARCH = "/api/public"
```

Comment:
Ah, I didn't test all cases, this endpoint change doesn't work:
`/api/public` returrns counts
`/api/beacon_search` returrns lists of ids, that was the point of the extra beacon code

Reply:
that's been changed in the latest katsu

Reply:
the same endpoint can return either depending on permissions

Reply:
public edge doesn't like katsu edge, so this will take me a moment or two...

Reply:
ah, okay, I had to hunt down the Bento repo branch, and then fight with it for a while. Working well now.